Enable UART on RaQ1

Unlike the current code suggests, the RaQ1 actually has an UART.  Only
the Qube1 (Qube 2700) lacks one.

Signed-off-by: Martin Michlmayr <tbm@cyrius.com>

--- a/arch/mips/cobalt/setup.c	2008-02-25 19:10:00.000000000 +0000
+++ b/arch/mips/cobalt/setup.c	2008-02-25 19:10:27.000000000 +0000
@@ -151,7 +151,7 @@
 #endif
 
 #ifdef CONFIG_SERIAL_8250
-	if (cobalt_board_id > COBALT_BRD_ID_RAQ1) {
+	if (cobalt_board_id >= COBALT_BRD_ID_RAQ1) {
 
 #ifdef CONFIG_EARLY_PRINTK
 		cobalt_early_console();

